packages/contracts/contracts/Passport.sol

```diff
@@ -118,4 +125,46 @@ contract Passport is ERC721, Ownable {
   receive() external payable {
       emit Received(msg.sender, msg.value);
   }
+
+  /**
+    * @dev A method to verify if the account belongs to the minter role
+    * @param account The address to verify.
+    * @return Return `true` if the account belongs to the minter role.
+    */
+  function isMinter(address account)
+      public
+      virtual
+      view
+      returns(bool)
+  {
+      return hasRole(MINTER_ROLE, account);
+  }
+
+  /**
+    * @dev Add accounts to the minter role. Restricted to admins.
+    * @param accounts The members to add as a member.
+    */
+  function addMinters(address[] memory accounts)
```

Comment:
I made both addMinters and removeMinters accept a list of address to prepare for the CSV bulk upload